source/CommonJobs/CommonJobs.Application.EvalForm/Commands/ChangeResponsibleCommand.cs

```diff
@@ -0,0 +1,84 @@
+﻿using CommonJobs.Application.Evaluations.EmployeeSearching;
+using CommonJobs.Domain.Evaluations;
+using CommonJobs.Infrastructure.RavenDb;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CommonJobs.Application.EvalForm.Commands
+{
+    public class ChangeResponsibleCommand : Command
+    {
+
+        public string _evaluatedUser { get; set; }
+
+        public string _period { get; set; }
+
+        public string _newResponsible { get; set; }
+
+        public ChangeResponsibleCommand (string evaluatedUser, string period, string  newResponsible)
+        {
+            _evaluatedUser = evaluatedUser;
+            _period = period;
+            _newResponsible = newResponsible;
+        }
+
+        public override void Execute()
+        {
+            var evaluationId = EmployeeEvaluation.GenerateEvaluationId(_period, _evaluatedUser);
+            var evaluation = RavenSession.Load<EmployeeEvaluation>(evaluationId);
+
+            if (evaluation == null)
+            {
+                throw new ApplicationException(string.Format("Error: Evaluación inexistente: {0}.", evaluationId));
+            }
+
+            // Check that the new responsable exist
+            var responsible = RavenSession
+                 .Query<Employee_Search.Projection, Employee_Search>()
+                 .Where(x => x.IsActive && x.UserName == _newResponsible).FirstOrDefault();
+
+            if (responsible == null)
+            {
+                throw new ApplicationException(string.Format("Error: Empleado inexistente: {0}.", _newResponsible));
+            }
+
+            // The only moment when a user cant change an evaluation responsible, is when the evaluation is complete and closed
+            if (evaluation.Finished)
+            {
+                throw new ApplicationException(string.Format("Error: This evaluation is closed. Responsible can't be changed."));
+            }
+
+            // Load the responsable calification
+            var responsibleCalification = RavenSession.Advanced.LoadStartingWith<EvaluationCalification>(evaluationId + "/").Where(x=>x.Owner==CalificationType.Responsible).FirstOrDefault();
```

Comment:
It seems to be the `old` responsible calification, could you make it _circumstance_ more explicit in the variable name?